bootstrap/config/config.go

```diff
@@ -310,6 +318,54 @@ func (cp *Processor) loadCommonConfig(
 	return nil
 }
 
+// loadCommonConfigFromUri will pull up to two separate common configs from the config provider
+func (cp *Processor) loadCommonConfigFromUri(
+	configFile string,
+	serviceConfig interfaces.Configuration,
+	serviceType string) error {
+
+	var err error
+
+	//TODO: implement reading from URI
+	commonConfig, err := cp.loadConfigYamlFromFile(configFile)
+	if err != nil {
+		return fmt.Errorf("could not load common configuration %s: %s", configFile, err.Error())
```

Comment:
error already has context
```suggestion
		return err
```